diff --git a/android/build.gradle b/android/build.gradle
index 4d48261b51f8a398f6512914a50a2a1b01ecfef6..cbcbd6a5df4ee2d5bdb3a95e241950f9bc92e30f 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -30,7 +30,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
