diff --git a/android/build.gradle b/android/build.gradle
index 52fbdad51afdc4b039bf05c1c08c76223d8cd6ed..99d499f59758900f381912e08c7ef69b2f4fe2ec 100644
--- a/android/build.gradle
+++ b/android/build.gradle
@@ -31,7 +31,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
